taskcluster/ffios_taskgraph/job.py

```diff
@@ -0,0 +1,130 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is a copy of https://github.com/mozilla-mobile/fenix/blob/34144fd254f00bd8a4176a296fad384ced69f2bf/taskcluster/fenix_taskgraph/job.py

I removed the `gradle` stuff. We don't need it on iOS. I kept the secret handling. I'll use taskcluster secrets soon.